_topics/en/simplicity.md

```diff
@@ -10,14 +10,31 @@ categories:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 excerpt: >
   **Simplicity** is a work in progress low level programming language designed
-  to offer greater functionality than Bitcoin Script. It will require a future
-  soft fork to be enabled on Bitcoin.
+  to offer greater flexibility and expressiveness than Bitcoin Script whilst
+  allowing you to verify the safety, security and costs of your program. It
+  offers native Merklized scripting, formal semantics and type checking.
+  It is not Turing complete but it can verify the execution of Turing complete
+  programs. Although Simplicity provides certain proofs of correctness,
+  sufficient care will need to be applied in the writing of Simplicity programs
+  to ensure they perform as intended. To use Simplicity directly on the
+  Bitcoin blockchain requires a soft fork and any such proposal is unlikely to
+  be formalized until at least 2022-23. Currently there is Simplicity support
```

Comment:
Nit: there's no need to use ranges with "at least"; just use the bottom of the range (e.g. "at least 2022").